modules/api/src/main/java/org/eclipse/xpanse/api/controllers/WorkFlowApi.java

```diff
@@ -67,31 +68,33 @@ public List<WorkFlowTask> queryTasks(
     /** Complete tasks based on task ID and set global process variables. */
     @Tag(name = "Workflow", description = "APIs to manage the Workflow")
     @Operation(description = "Complete tasks by task ID and set global process variables .")
-    @PutMapping(value = "/workflow/complete/task/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @PutMapping(
+            value = "/workflow/complete/task/{taskId}",
+            produces = MediaType.APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.OK)
-    @AuditApiRequest(methodName = "getCsFromWorkflowTaskId")
+    @AuditApiRequest(methodName = "getCsFromWorkflowTaskId", paramTypes = UUID.class)
     public void completeTask(
```

Comment:
The taskId in workflow is not a UUID. Just keep type String

Reply:
updated.